main.go

```diff
@@ -17,14 +17,34 @@ limitations under the License.
 package main
 
 import (
+	"context"
+	"net/url"
 	"os"
 
+	"github.com/kubescape/go-logger"
+	"github.com/kubescape/go-logger/helpers"
+	"github.com/kubescape/storage/pkg/cmd/server"
+	"github.com/kubescape/storage/pkg/config"
 	genericapiserver "k8s.io/apiserver/pkg/server"
 	"k8s.io/component-base/cli"
-	"github.com/kubescape/storage/pkg/cmd/server"
 )
 
 func main() {
+	ctx := context.Background()
+	c, err := config.LoadConfig("/etc/config")
```

Comment:
question: this would mean that the storage would need to have this config file present?

How would this be mapped in the Helm chart? How would we ensure this config exists? Should we provide an example?

Reply:
either we need the file present, or having some env set in order to know the customerID to enrich spans
https://github.com/kubescape/helm-charts/pull/201